src/.nuget/packageLoad.targets

```diff
@@ -143,7 +137,11 @@
     <Message Importance="High" Text="Restoring in $(ProjectDirectory)" />
 
     <!-- Restore packages for this directory -->
-    <Exec Command="$(DnuRestoreCommand) &quot;$(ProjectDirectory.TrimEnd('\').TrimEnd('\'))&quot; --quiet" />
+    <!--
+    <Exec Command="$(NugetRestoreCommand) $(NuGetPackageRestoreOptions) -PackagesDirectory &quot;$(HOME)/.nuget/packages&quot; &quot;@(ProjectJsonFile)&quot;" />
+    <Exec Command="$(DnuToolPath) restore &quot;@(ProjectJsonFile)&quot; - -parallel " />
+    -->
+    <Exec Command="$(DnuRestoreCommand) &quot;@(ProjectJsonFile)&quot;" />
```

Comment:
Will do. BTW, only one of the comments is stale -- the DnuRestoreCommand should go away and NugetRestoreCommand should be used eventually, that's why I left it there.